### PR TITLE
SRE-2515: New LICENSE file as expected by OpenSSF Scorecard

### PR DESCRIPTION
**Pull request by osalyk**
Doc-only: true

### CODE REVIEW

**Review by grom72 — DISMISSED** [review]

**Review by daltonbohning — DISMISSED** [review]

File: Makefile [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+# Copyright (C) 2019-2024 Intel Corporation
```
Comment (janekmi) [inline Makefile:2]:
```suggestion
# Copyright (c) 2018-2024 Intel Corporation
```

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,54 @@
+BSD-2-Clause Plus Patent License
+
+Copyright 2019-2024 Intel Corporation.
```
Comment (janekmi) [inline LICENSE:3]:
```suggestion
Copyright 2018-2024 Intel Corporation.
```

**Review by janekmi — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (osalyk) [reply Makefile]: Done.

File: LICENSE (hunk 2 above)
Comment (osalyk) [reply LICENSE]: Done.

**Review by grom72 — APPROVED** [review]

**Review by janekmi — APPROVED** [review]

**Pull request merged**